config/default/field.field.node.government_gazette.field_gazette_issue_number.yml

```diff
@@ -0,0 +1,23 @@
+uuid: 2c12c469-d072-486c-b0a7-842716ceecd8
+langcode: en
+status: true
+dependencies:
+  config:
+    - field.storage.node.field_gazette_issue_number
+    - node.type.government_gazette
+id: node.government_gazette.field_gazette_issue_number
+field_name: field_gazette_issue_number
+entity_type: node
+bundle: government_gazette
+label: 'Gazette issue number'
+description: ''
+required: false
+translatable: false
+default_value: {  }
+default_value_callback: ''
+settings:
+  min: null
+  max: null
+  prefix: ''
+  suffix: ''
+field_type: integer
```

Comment:
This field need to be a string. Sometimes a gazette number will be `bis` or `special` or `12A` or some other weird things.

Reply:
I've corrected this in the spreadsheet.